### PR TITLE
Update 0013-manage-features.md

### PR DESCRIPTION
**Pull request by sushmita-m**
Testing with imagasfare
Helpful resources

Screenshot 2020-12-23 at 11 07 48 AM

### CODE REVIEW

**Comment by sushmita-m** [comment]
testign comment

**Comment by sushmita-m** [comment]
test

File: 0013-manage-features.md [hunk 1]
```diff
@@ -29,6 +29,7 @@
 789
 0009
 9023
+asdfadf
```
Comment (sushmita-m) [inline 0013-manage-features.md:32]: afads

**Review by suana — COMMENTED** [review]
sdgsdg

**Review by suana — APPROVED** [review]
afafa

**Review by suana — APPROVED** [review]

**Comment by sushmita-m** [comment]
test

**Pull request closed without merging**